app/models/cid.rb

```diff
@@ -0,0 +1,127 @@
+class Cid
+  include Neo4j::ActiveNode
+
+  property :documentType
+  property :contentID
+  property :name
+  property :description
+  property :pagerank
+  property :title
+
+  has_many :out, :eligibilities, type: :HAS_ELIGIBILITY, model_class: :Eligibility
+
+  def self.where_not_ineligible(attributes)
+    Eligibility.
+      all.
+      reject { |eligibility| eligibility.ineligible?(attributes) }.
+      map { |eligibilty| eligibilty.cids }.
+      flatten.
+      compact
+  end
+
+  def self.test_toy_example
+    # This is one way in which we might make the 16-19 bursary fund (https://www.gov.uk/1619-bursary-fund/eligibility) work
+    Eligibility.delete_all
+    StudyRequirement.delete_all
+    AgeRequirement.delete_all
+    ResidencyRequirement.delete_all
+    OrRequirementGroup.delete_all
+    AndRequirementGroup.delete_all
+    Cid.find_by(contentID: "f4b96a38-5247-4afd-b554-8a258a0e8c93").eligibilities.delete_all
+    Cid.find_by(contentID: "33f9d8a1-2a3c-4b71-abda-b67b38bf0d67").eligibilities.delete_all
+
+    eligibility = Eligibility.create()
+    content_page = Cid.find_by(contentID: "f4b96a38-5247-4afd-b554-8a258a0e8c93")
+    content_page.eligibilities << eligibility
+
+    base_requirement = AndRequirementGroup.create()
+    eligibility.requirement = base_requirement
+
+    # be at least 16 and under 19 on 31 August 2019
+    age_requirement = AgeRequirement.create(min_age: 16, max_age: 18)
```

Comment:
Perhaps a bit pedantic with language here, but is _under 19_ technically the same as `18 <= 18`? (for example, if you're nearly 19, you're over 18y 0m).